webtranslate/__main__.py

```diff
@@ -3,7 +3,7 @@
 
 from . import bottle
 from . import main
-from .click import click_additional_options
+from .click_extras import click_additional_options
```

Comment:
On the other repositories I marked this error as "won't fix", as CodeQL is wrong here. `.click` and `click` in Python3 are completely different packages, so the error itself is not correct. One can, however, debate that the naming can be confusing. But if we go this route, we might as well fix it for all the other repositories too :)

Reply:
Mm, that's probably the better solution